### ISSUE TITLE
No predefined value in calendar

### ISSUE THREAD

**Issue by martinpesout** [Adphorus/react-date-range#52]
Is there any chance to define no default value when displaying calendar? I want to use this plugin only for selecting ranges, but I want to display only blank calendars by default after first render.

**Comment by ZachPerkitny** [comment]
Did you ever find a solution ?

**Comment by martinpesout** [comment]
Exactly. I think that currently there is no solution for this problem

**Comment by ZachPerkitny** [comment]
@martinpesout 

Hopefully they'll make this possible, or maybe I'll make a pull request.

**Comment by jacobrask** [comment]
If a calendar state without a date selected is valid should you be able to unselect the date as well, or is that something you'd implement yourself?

**Comment by krupasavadiya** [comment]
Hello,

is there any update on this issue?

Regards

**Comment by hagealex** [comment]
Hello,

I have the same need. Is there any update on this?

Regards
